executor/executor_test.go

```diff
@@ -4139,6 +4139,18 @@ func (s *testSuiteP1) TestSplitRegion(c *C) {
 	tk.MustExec("create table t1(a int, b int)")
 	tk.MustQuery("split table t1 between(0) and (10000) regions 10;").Check(testkit.Rows("9 1"))
 	tk.MustQuery("split table t1 between(10) and (10010) regions 5;").Check(testkit.Rows("4 1"))
+
+	// Test split region for partition table.
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t (a int,b int) partition by hash(a) partitions 5;")
+	tk.MustExec("split table t between (0) and (1000000) regions 5;")
```

Comment:
Do we need to check the result like line 4141?

Reply:
done.